backend/repositories/ContentRepository.ts

```diff
@@ -23,13 +41,12 @@ export type ContentSearchParametersType = {
   blockFrom?: number;
   blockTo?: number;
   contentHash?: string;
+  relatedContentHash?: string;
 };
 
-type ResponseAnnouncementResponse = AnnouncementResponse & { announcement: ReplyAnnouncement | ReactionAnnouncement };
-
-// TODO: more tightly bind the announcement types. Reactions and replies only in contentResponseMap.
-const contentMap = new Map<string, AnnouncementResponse>();
-const contentResponseMap = new Map<string, ResponseAnnouncementResponse>();
+type RelatedAnnouncementResponse = AnnouncementResponse & {
+  announcement: ReplyAnnouncement | ReactionAnnouncement | TombstoneAnnouncement | UpdateAnnouncement;
+};
```

Comment:
I'm using this type in my PR too. Can we put it somewhere more global and export it?

Same consideration with naming.

Reply:
Feel free to move it to `backend/types/types.ts` and export it from there. Same response on the naming.

Reply:
Moved.